frontend/src/components/PlannerChart.tsx

```diff
@@ -27,7 +27,14 @@ import { apiV2 } from '../api/api';
 
 const classes = classNames.bind(css);
 
-const colors = ['blue', 'red', 'green', 'orange'] as const;
+const colors = [
+  '#E24A4A',
+  '#53D4F0',
+  '#8CE434',
+  '#AA75EE',
+  '#FBD92A',
+  '#EA9128',
```

Comment:
Maybe these should be added to colors.module.scss

Reply:
And noting just to remember what was discussed in weekly: the values can be darkened